packages/ag-charts-enterprise/src/series/treemap/treeMapLabelFormatter.test.ts

```diff
@@ -0,0 +1,218 @@
+import { _Scene } from 'ag-charts-community';
+
+import {
+    formatSingleLabel,
+    formatStackedLabels,
+    generateLabelSecondaryLabelFontSizeCandidates,
+    maximumValueSatisfying,
+} from './treeMapLabelFormatter';
+
+const { Text } = _Scene;
+
+const wrap = jest.fn();
+const computeBBox = jest.fn();
+
+Text.wrap = wrap;
+Text.prototype.computeBBox = computeBBox;
+
+afterEach(() => {
+    jest.restoreAllMocks();
+});
+
+describe('generateLabelSecondaryLabelFontSizeCandidates', () => {
+    it('creates a font scale', () => {
+        expect(
+            generateLabelSecondaryLabelFontSizeCandidates(
+                // @ts-expect-error Fix typechecking here
```

Comment:
Can we resolve these `tsc` bypasses, we've avoid the need to do this so far except for a couple of special cases?